recipe/meta.yaml

```diff
@@ -31,12 +31,13 @@ test:
     - xmlwf -h
 
 about:
-  home: http://expat.sourceforge.net/
+  home: https://expat.sourceforge.net/
```

Comment:
This URL is no longer valid; it should be updated to the GitHub or GitHub pages (or wherever it redirects to now).

Reply:
The original URL `http://expat.sourceforge.net/` remains valid and is up to date. It seems sourceforge just doesn't provide access via https?

Reply:
The home url was set to `http` since `https` is not supported for that url at this point.

Reply:
Interesting; on my internet, the HTTPS SourceForge URL results in a connection refused, while the HTTP SourceForge URL redirects to https://libexpat.github.io/.

Reply:
the `home` url was updated
https://libexpat.github.io